.github/workflows/test-control-server.yml

```diff
@@ -47,6 +47,8 @@ jobs:
           echo "DEBUG=False" >> .env
           echo "ID_KEY=some secret key" >> .env
           echo "BEHIND_PROXY=False" >> .env
+          echo "TRUSTED_PROXIES=172.17.0.0/16,172.18.0.0/16" >> .env
+          echo "ADMIN_KEY=RSU7CM9hbZ4swfHW0eVppblG4EtYiiVcREcYhn3SbraZXcfL3lP0bbDF2cpo4jUA" >> .env
```

Comment:
Is it ok to include the key in the repository when it is public?

Reply:
It was a hard coded key just to run the tests but I have removed it now and we can create a new key for authorization.